docs/index.md

```diff
@@ -10,6 +10,10 @@ Mock also offers a multi-package command (`--chain`), that can build chains of p
 
 Mock is capable of building SRPMs from source configuration management if the `mock-scm` package is present, then building the SRPM into RPMs. See `--scm-enable` in the documentation.
 
+![Fedora package](https://img.shields.io/fedora/v/mock/rawhide)
+
+![Fedora package](https://img.shields.io/fedora/v/mock-core-configs/rawhide)
```

Comment:
This looks confusing since the package names aren't in the badge titles.

Reply:
I am confused because I do not understand what exactly you mean by badge titles. :)

Reply:
It would be better to have:
`mock: <the badge>`
`mock-core-configs`: <the badge>`
Because the badge itself doesn't claim what package it belongs to. Just "Fedora: v36.3".